SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalanceEnforcer/SpatialLoadBalanceEnforcerTest.cpp

```diff
@@ -45,71 +45,28 @@ void AddEntityToStaticComponentView(USpatialStaticComponentView& StaticComponent
 	}
 }
 
-void InitialiseVirtualWorkerTranslator(SpatialVirtualWorkerTranslator* VirtualWorkerTranslator, bool bAddDefaultWorkerMapping)
+void InitialiseVirtualWorkerTranslator(SpatialVirtualWorkerTranslator* VirtualWorkerTranslator)
```

Comment:
We can get rid of this function. Merge it into CreateVirtualWorkerTranslator

Reply:
done